### DISCUSSION TITLE
I want to use 'STM32Duino' through 'STM32L151C8'.

### DISCUSSION THREAD

**Discussion by SeongJongKwak** [stm32duino/Arduino_Core_STM32#2396] [General]
I want to use 'STM32Duino' through 'STM32L151C8'.

I'm curious about how. help.
1. First, how should I configure the circuit? There is no internet information.
 - Circuit-wise, it looks the same as ‘STM32L151C8T’ and ‘STM32F103C8’. Can I make the circuit diagram the same as before? Will it work without problem?

2. Can boot loader use ‘generic_boot20_pc13’?
 - How should I install the bootloader for 'STM32L151C8'?

help me.

**Comment by fpistm** [comment]
Please avoid to open several topics for the same things, this is not the first time.
https://www.stm32duino.com/viewtopic.php?t=2415

You already been warned, next time I will have to block.

**Comment by SeongJongKwak** [comment]
@fpistm 
I'm sorry. I won't make that mistake next time. 
I didn't know there was a rule.
I'm really sorry.
